test/suite/hook_disagg.py

```
@@ -94,6 +94,9 @@ def wiredtiger_open_replace(orig_wiredtiger_open, homedir, curconfig):
     if 'tiered_storage=' in curconfig:
         skip_test("cannot run disagg hook on a test that uses tiered_storage in the config string")
 
+    if 'log=(enabled)' in curconfig or 'log=(enabled=true)' in curconfig:
+        skip_test("Log tables are not supported in disagg. FIXME-WT-15221 Should throw an error when this is set in disagg")
```

The `FIXME-WT-15221 Should throw an error when this is set in disagg` should be placed in a comment, since the s_all scripts scan comments to find Fix me markers and report any that are still present in the code.

---

done